lib/validates_zipcode/cldr_regex_collection.rb

```diff
@@ -196,7 +196,7 @@ module CldrRegexpCollection
       LI: /\A\d{4}\z/,
       LV: /\A([a-zA-Z]|\d){3,8}\z/,
       LY: /\A\d{5}\z/,
-      LT: /\A([a-zA-Z]){2}(-)\d{4,5}\z/,
+      LT: /\A(([a-zA-Z]){2}(-))?\d{4,5}\z/,
```

Comment:
This is a bit broad, given the format options: https://en.wikipedia.org/wiki/Postal_codes_in_Lithuania

Also quite different from the regexp you link in the issue. 

What about: `/\A(LT-)?\d{4,5}\z/` ?

Also, the 4 digit format is the old Soviet Union format, unused since 2005. This library wasn't around until much later (or was compatible with software from 2005), so I think we can safely deprecate and remove that format without breaking any app using the gem. I'm not from LT though, so I don't know how often could that format be used now? 

If that is the case, what about `/\A(LT-)?\d{5}\z/`?

Reply:
Yep, agree about removing the 4 digit option since it is now 14+ years old. If someone submits an issue, easy enough to add it back. I'm also not from LT, we encountered an issue with this because we have customers in LT and were running into an invalid zip code error with NNNNN.